### PR TITLE
DateRangeBox - startDate/endDate options should have 'default' prefix in React

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by ksercs — APPROVED** [review]

File: js/ui/date_range_box.d.ts [hunk 1]
```diff
@@ -93,6 +93,7 @@ export type Properties = Omit<DateBoxBaseOptions<dxDateRangeBox>, 'inputAttr' |
      * @docid dxDateRangeBoxOptions.endDate
      * @default null
      * @public
+     * @fires BaseWidgetOptions.onValueChanged
```
Comment (ksercs) [inline js/ui/date_range_box.d.ts]:
```suggestion
     * @fires EditorOptions.onValueChanged
```

**Review by ksercs — CHANGES_REQUESTED** [review]

**Review by ksercs — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

File: js/ui/date_range_box.d.ts [hunk 2]
```diff
@@ -93,6 +93,7 @@ export type Properties = Omit<DateBoxBaseOptions<dxDateRangeBox>, 'inputAttr' |
      * @docid dxDateRangeBoxOptions.endDate
      * @default null
      * @public
+     * @fires
```
Comment (IlyaKhD) [inline js/ui/date_range_box.d.ts]:
```suggestion
     * @fires dxDateRangeBoxOptions.onOptionChanged
```

File: js/ui/date_range_box.d.ts [hunk 3]
```diff
@@ -161,6 +162,7 @@ export type Properties = Omit<DateBoxBaseOptions<dxDateRangeBox>, 'inputAttr' |
      * @docid dxDateRangeBoxOptions.startDate
      * @default null
      * @public
+     * @fires
```
Comment (IlyaKhD) [inline js/ui/date_range_box.d.ts]:
```suggestion
     * @fires dxDateRangeBoxOptions.onOptionChanged
```

File: js/ui/date_range_box.d.ts (hunk 2 above)
Comment (IlyaKhD) [reply js/ui/date_range_box.d.ts]: `@fires` can not be empty

**Review by IlyaKhD — CHANGES_REQUESTED** [review]

**Review by mpreyskurantov — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**